index.html

```diff
@@ -885,6 +856,158 @@ <h5>Timing Properties</h5>
+                    <td><code>visual;text;location</code></td>
+                    <td>yes</td>
+                  </tr>
+                </tbody>
+              </table>
+              <p>For example, in this table A could be one of the values listed in <a>Script Represents</a> property,
+                and B could be the value of a <a>Represents</a> property.</p>
+            </aside>
+            <p>The permitted values for <code>&lt;<a>content-descriptor</a>&gt;</code>
+              are either those listed in the following <a>registry table</a>, or can be user-defined:</p>
+              <div class="registry-table-section">
+                <table class="data">
+                  <caption><a>Registry table</a> for the <code>&lt;<a>content-descriptor</a>&gt;</code> component
+                    whose Registry Definition is at <a href="#registry-table-content-descriptor"></a></caption>
+                  <thead>
+                    <th><code>&lt;<a>content-descriptor</a>&gt;</code></th>
+                    <th>Status</th>
+                    <th>Description</th>
+                    <th>Example usage</th>
+                    <th>Notes</th>
+                  </thead>
+                  <tbody>
+                    <tr>
+                      <td><code>audio</code></td>
+                      <td>Provisional</td>
+                      <td>Audio content</td>
+                      <td>Dubbing, translation and hard of hearing subtitles and captions, pre- and post- production scripts</td>
+                      <td>See sub-types.</td>
+                    </tr>
+                    <tr>
+                      <td><code>audio;dialogue</code></td>
+                      <td>Provisional</td>
+                      <td>Verbal communication, in audio</td>
+                      <td>Dubbing, translation and hard of hearing subtitles and captions, pre- and post- production scripts</td>
+                      <td>For example, a spoken conversation.</td>
+                    </tr>
+                    <tr>
+                      <td><code>audio;nonDialogueSounds</code></td>
+                      <td>Provisional</td>
+                      <td>Sounds that are not verbal communication, in audio</td>
+                      <td>Translation and hard of hearing subtitles and captions, pre- and post- production scripts</td>
+                      <td>For example, significant sounds, such as a door being slammed in anger.</td>
+                    </tr>
+                    <tr>
+                      <td><code>visual</code></td>
+                      <td>Provisional</td>
+                      <td>Parts of the visual image</td>
+                      <td>Audio Description</td>
+                      <td>See sub-types.</td>
+                    </tr>
+                    <tr>
+                      <td><code>visual;dialogue</code></td>
+                      <td>Provisional</td>
+                      <td>Verbal communication, in sign language</td>
```

Comment:
```suggestion
                      <td>Indicates that the DAPT content represents verbal communication, in sign language, within the visual of the programme</td>
```

Reply:
And similar changes for the next rows...

Reply:
What is "the visual of the programme"?

Reply:
Changing to "visual image of the programme"

Reply:
Done in 09b070e